src/components/contentGetters/services/category.service.js

```diff
@@ -52,17 +66,32 @@ angular.module('znk.infra.contentGetters').service('CategoryService',
             });
         };
 
-        self.getCategoryLevel1ParentById = function (categoryId) {
+        self.getCategoryLevel1ParentById = function (categoryId, sync) {
```

Comment:
Create new functions with the postfix "sync" instead of managing both in the existing functions